src/medUtilities/medUtilities.cpp

```diff
@@ -176,7 +176,26 @@ void medUtilities::switchToVolumeRendering(medAbstractView *view)
                 break;
             }
         }
-        mode3DParam->setValue("VR");
+
+        switch(mode3D)
+        {
+        case VR:
+            mode3DParam->setValue("VR");
+            break;
+        case MIP_MAXIMUM:
+            mode3DParam->setValue("MIP - Maximum");
+            break;
+        case MIP_MINIMUM:
+            mode3DParam->setValue("MIP - Minimum");
+            break;
+        case MSR:
+            mode3DParam->setValue("MSR");
+            break;
+        default:
```

Comment:
You don't need a default here, it's not possible since you took into account all the possible values (the argument type being Mode3DType, you can't call this function with a value not in the enum)